packages/frontend-shared/src/gql-components/topnav/TopNav.vue

```diff
@@ -386,4 +414,8 @@ onKeyStroke(' ', (event) => {
   resetPrompt(event)
 })
 
+onKeyStroke('Escape', (event) => {
```

Comment:
Comment above: `// using onKeyStroke twice as array of keys is not supported till vueuse 6.6:`. We use `7.2.2` now, can we make this simpler?